test/test_other.py

```diff
@@ -319,6 +319,7 @@ def test_emcc_output_worker_mjs(self, args):
     src = read_file('subdir/hello_world.mjs')
     self.assertContained("new URL('hello_world.wasm', import.meta.url)", src)
     self.assertContained("new Worker(new URL('hello_world.worker.js', import.meta.url), {type: 'module'})", src)
```

Comment:
I wonder if we should call this file `.mjs` if we are going to load it like this?  Would that remove the need for the explicit `type`?

Reply:
No, browsers don't care about file extensions.

Also, unlike for the main output, we don't really intend to use any ESM-specific features - it has to be backward-compatible with older browsers that ignore `{type:'module'}` and load it as regular script Worker. Module Workers are relatively new feature compared to regular modules, so for now I'd keep them as regular scripts, just with this extra hint for bundlers and newer runtimes that don't support regular scripts anymore.